Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.ts

```diff
@@ -10,7 +10,13 @@ import { WindowRef } from 'src/app/services/windowRef.service';
 import { ColumnDefinitions } from 'src/app/shared/config/columnDefinitions';
 import { Person, PersonStatus } from 'src/app/shared/models';
 import { BookmarksState, ToggleBookmark } from 'src/app/shared/states';
+import { MatDialogConfig, MatDialog, MAT_DIALOG_DATA } from '@angular/material';
+import { IncorrectUserInformationComponent } from 'src/app/shared/dialogs/user-information/incorrect-user-information.component';
 
+export interface DialogData {
```

Comment:
Still the name, if we name everything like this nobody will know which DialogData he is referencing to: `IncorecctUserInfromationDialogData`